deploy/install/02-components/05-webhook.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: longhorn-webhook
+  name: longhorn-webhook
+  namespace: longhorn-system
```

Comment:
Is it better to add podAntiAffinity here?

Reply:
Good suggestion. I added the `preferredDuringSchedulingIgnoredDuringExecution`.